### DISCUSSION TITLE
Release v0.8.0-beta

### DISCUSSION THREAD

**Discussion by MindStudioOfficial** [MindStudioOfficial/ndiscopes#13] [Announcements]
# Features/Improvements
- 🚀 Improved mask performance
- 🔳 Mask now works on stills
- 🔳 Mask background now stays visible
- 🎨 Added colorspace differentiation for vectorscope and CIE chromaticity scope
- 🚀 Improved window animations for settings, metadata, and colorfilters
-  ↕ Zoomed in the CIE 1931 scope
- 🎨 Added rec601 colorspace to CIE 1931 scope
- 📏 Vectorscope lines now correspond to the primary and secondary color vectors
# Bug Fixes
- 🐞 Fixed #9 metadata table selection not working correctly
- 🐞 Fixed wrong calculation of vectorscope squares

<hr /><em>This discussion was created from the release <a href='https://github.com/MindStudioOfficial/ndiscopes/releases/tag/'>Release v0.8.0-beta</a>.</em>

**Comment by sisoyjuan** [comment]
I have a computer with Nvidia P1000. I understand it is compatible but I have got an error "GPU not compatible". Am I wrong?

Regards

**Reply by MindStudioOfficial** [reply]
Your card has a Compute Capability (Version) of 6.1 which should be supported. I have tested the software with GPUs of the 10 series which have the same CC and got no errors. Please make sure that your drivers are up to date since this can also cause problems. It would also be interesting if you could test other versions of NDIScopes, for example version v0.7.1-beta since this was compiled before I upgraded my CUDA environment. 
If your problem is not resolved please open a new Issuein the "[Issues](https://github.com/MindStudioOfficial/ndiscopes/issues/new/choose)" section so I can track this problem further.

**Comment by sisoyjuan** [comment]
Updated GPU graphic drivers and working as expected. Thanks friend

**Comment by DeeJayMX** [comment]
Is there a TCP or (better) 1 a HTTP API to remote control and Switch source from an external controller (Bitfocus Companion)?

Thanks

**Reply by MindStudioOfficial** [reply]
Not yet, but it is a feature that I am considering. Maybe open a new Feature Request [here](https://github.com/MindStudioOfficial/ndiscopes/issues/new/choose) with the exact things you want to control so I can start implementing this feature.

**Comment by DeeJayMX** [comment]
And Is there an option to change the layout, I don't mind to see the composite output but prefer to have the waveform in the big window.

Although your soft looks really great !

Thanks

**Reply by MindStudioOfficial** [reply]
For now the layout is fixed but I am thinking of adding some sort of "fullscreen scopes" mode.
